### ISSUE TITLE
texts from `describe` and `it` blocks are not printed when using browser `nightmare`

### ISSUE THREAD

**Issue by jirikrepl** [meteortesting/meteor-mocha#102]
Hi, after update to meteor from 1.8.1 to 1.10.2 and updates:

`meteortesting:mocha@1.1.2`	-> `meteortesting:mocha@2.0.0`
and internal update:
`meteortesting:mocha-core@1.0.1`	`meteortesting:mocha-core@7.0.1`

The output of the client-side tests is not visible if `TEST_BROWSER_DRIVER=nightmare`
I was able to reproduce this issue in this repository if you do `meteor npm t`
https://github.com/jirikrepl/meteortesting-mocha-client-reporter-bug-reproduction

Here's the screenshot from the terminal output:
My test file runs both on the server and client-side. You can see a lot of prints on the server-side ('some test', 'can do something'), but only `%s%s` on client-side:

The error and warnings: 
```
[ERROR] There is no route for the path: /
%cElectron Security Warning (Insecure Resources)
%cElectron Security Warning (Insecure Content-Security-Policy)
```
shouldn't be related to the reporter "print" issue. In my app, I don't see these errors, but still cannot see the content of "describe" and "it" block strings

<img width="1536" alt="Screenshot 2020-05-03 at 13 04 22" src="https://user-images.githubusercontent.com/2588479/80912711-33383880-8d3f-11ea-804c-b00561293134.png">

**Comment by SimonSimCity** [comment]
Thanks for the detailed report. Please run the tests again using `Puppeteer` (https://github.com/meteortesting/meteor-browser-tests#puppeteer). If that works, this bug needs to be reported to the https://github.com/meteortesting/meteor-browser-tests repository.

**Comment by jirikrepl** [comment]
Great! I did `npm i --save-dev puppeteer` and added `TEST_BROWSER_DRIVER=puppeteer` to my package.json test scripts. Client reporter string are now back and tests are passing. Thanks!

**Comment by SimonSimCity** [comment]
Please test with the latest version. `meteortesting:browser-tests` has just been released as `1.3.4` which should fix this bug. If it doesn't, please open a ticket in https://github.com/meteortesting/meteor-browser-tests.
